docker/config/application.yaml

```diff
@@ -31,14 +31,14 @@ fhirdata:
   # There is no Questionnaire in our test FHIR server, but it is added to
   # prevent regression of https://github.com/google/fhir-data-pipes/issues/785.
   # TODO: add resource table creation to e2e tests.
-  resourceList: "Patient,Encounter,Observation,Questionnaire,Condition,Practitioner,Location,Organization"
+  resourceList: "Patient,Encounter,Observation,Questionnaire,Condition,Practitioner,Location,Organization,DiagnosticReport,Immunization,MedicationRequest,PractitionerRole,Procedure"
   numThreads: -1
   autoGenerateFlinkConfiguration: true
   createHiveResourceTables: true
   thriftserverHiveConfig: "config/thriftserver-hive-config_local.json"
   hiveResourceViewsDir: "config/views"
   # structureDefinitionsDir: "config/profile-definitions"
-  structureDefinitionsClasspath: "/r4-us-core-definitions"
+  # structureDefinitionsClasspath: "/r4-us-core-definitions"
```

Comment:
With the latest changes from this [PR](https://github.com/google/fhir-data-pipes/pull/986) , I am assuming the sample Observation flat view should not fail, as it considers all the multiple choice types from all extensions (assuming it was failing due to some restrictions on the choice types earlier).

Reply:
Okay, I'll wait for that PR to be merged and probably uncomment this line.

Reply:
The [PR](https://github.com/google/fhir-data-pipes/pull/986) has been merged, you can uncomment this and retest the scenario

Reply:
Great, I just reverted this change.